tests/api_test.py

```diff
@@ -1919,6 +1919,20 @@ def f(x1, x2, g):
     ):
       jax.vmap(f, (0, 0, None))(jnp.ones(2), jnp.ones(3), jnp.add)
 
+  def test_vmap_inconsistent_sizes_constructs_proper_error_message_kwargs(self):
+    def f(x1, x2, a3):
```

Comment:
We typically add cross-references to the solved issue for tests like this:
```suggestion
    # regression test for https://github.com/jax-ml/jax/issues/24406
    def f(x1, x2, a3):
```

Reply:
Done.